pom.xml

```diff
@@ -67,6 +67,11 @@
             <artifactId>structured-logging</artifactId>
             <version>${structured-logging.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.googlejavaformat</groupId>
+            <artifactId>google-java-format</artifactId>
+            <version>1.24.0</version>
```

Comment:
We should define this version number as a property and then reference it here, like for all the other dependencies (e.g. structured-logging).